src/com/nutrons/stronghold/Turret.java

```diff
@@ -0,0 +1,58 @@
+package com.nutrons.stronghold;
+
+import com.nutrons.framework.Subsystem;
+import com.nutrons.framework.controllers.ControllerEvent;
+import com.nutrons.framework.controllers.FollowerTalon;
+import com.nutrons.framework.controllers.LoopPropertiesEvent;
+import com.nutrons.framework.subsystems.Settings;
+import com.nutrons.framework.util.FlowOperators;
+import io.reactivex.Flowable;
+import io.reactivex.functions.Consumer;
+
+public class Turret implements Subsystem{
+    private final Flowable<Double> angle;
+    private final Consumer<ControllerEvent> hoodMaster;
+    private final FollowerTalon hoodSlave;
+    private final Flowable<ControllerEvent> PIDControllerMotors;
+
+    private final Consumer<ControllerEvent> shooter;
+    private Flowable<ControllerEvent> PIDControllerShooter;
+    private Flowable<Double> MOTOR_POWER;
+
+    private Settings PIDSettings;
+    private final double initial = 0.0; //If the PID settings doesn't emit any values, then it defaults to 0.0
+
+    private Flowable<Double> arcLength;
+    private static final double HOOD_RADIUS_IN = 10.5;
+
+    Turret(Flowable<Double> angle, Consumer<ControllerEvent> master, FollowerTalon slave,
+           Flowable<Boolean> triggerValues, Consumer<ControllerEvent> shooter){
+        this.angle = angle;
+        this.hoodMaster = master;
+        this.hoodSlave = slave;
+        arcLength = this.angle.map( x -> (x*2*Math.PI*HOOD_RADIUS_IN) / (360) );
```

Comment:
This isn't very readable.  It could be improved by adding spaces between the arguments, moving the `HOOD_RADIUS_IN` next to the `* 2` and removing the extra parens.  You may want to add a comment to explain what is happening here.  `(x* Math.PI * (2 * HOOD_RADIUS_IN)) / 360`